pkg/controller/master/controller.go

```diff
@@ -26,6 +25,7 @@ var registerFuncs = []registerFunc{
 }
 
 func register(ctx context.Context, server *server.Server, management *config.Management) error {
+	indexeres.RegisterManagementIndexers(management)
```

Comment:
append a line separator here since the `registerFuncs` is a new block of code.

Reply:
fixed